### ISSUE TITLE
Graphql-Pro: Deprecation warning with Redis 4.8.0

### ISSUE THREAD

**Issue by sealabcore** [rmosolgo/graphql-ruby#4167]
When using Graphql-Pro 1.23.0 with Redis 4.8.0 this deprecation warning starts to show.

```
Redis#sadd will always return an Integer in Redis 5.0.0. Use Redis#sadd? instead.(called from: /Users/heyyyy/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/graphql-pro-1.23.0/lib/graphql/pro/subscriptions/redis_storage.rb:196:in `block (2 levels) in write_subscription')
```

**Comment by rmosolgo** [comment]
Thanks for the heads-up! I'll update the library and follow up here soon.

**Comment by rmosolgo** [comment]
Thanks again for reporting this. I just updated the gem's use of `sadd` and `srem` to be backwards-compatible with redis 4.x _and_ forwards-compatible with forthcoming redis 5.x. The updates are released in graphql-pro 1.23.1 -- please let me know if you notice anything else after updating to that version!
